### PR TITLE
Client: Fix invisible gauss beam

### PR DESCRIPTION
**Pull request by rtxa**
Fix issue from #14, it's a side-effect from the merged pull request I did for damage cvars.

Fixed by @tmp64

### CODE REVIEW

File: dlls/gauss.cpp [hunk 1]
```diff
@@ -314,12 +314,19 @@ void CGauss::StartFire( void )
...
+#endif
 	}
 	else
 	{
+#ifdef CLIENT_DLL
+		flDamage = 200 * (( gpGlobals->time - m_pPlayer->m_flStartCharge) / GetFullChargeTime() );
+#else
 		flDamage = gSkillData.plrDmgGaussSecondary * (( gpGlobals->time - m_pPlayer->m_flStartCharge) / GetFullChargeTime() );
-	}
+#endif
```
Comment (tmp64) [inline dlls/gauss.cpp:329]: I think a `}` was lost on the line just after `#endif`

Comment (rtxa) [reply dlls/gauss.cpp:329]: Yeah, I had solved this before your comment, but thanks anyway.

**Comment by tmp64** [comment]
This is actually a client fix. Weapons source code is included in both server and client for weapon prediction.

**Comment by LevShisterov** [comment]
Closed via e0ecdffe7914373c2e33d98611fefe3964cee8d0

**Pull request closed without merging**